marine-js/npm-package/src/__test__/logging.spec.ts

```diff
@@ -0,0 +1,78 @@
+import fs from 'fs';
+import path from 'path';
+import { MarineService } from '../MarineService';
+import { LogLevel } from '../types';
+
+const examplesDir = path.join(__dirname, '../../../../examples');
+
+const loadWasmModule = async (waspPath: string) => {
+    const fullPath = path.join(waspPath);
+    const buffer = await fs.promises.readFile(fullPath);
+    const module = await WebAssembly.compile(buffer);
+    return module;
```

Comment:
"module" is sometimes a reserved keyword but also you can just do
```ts
return WebAssembly.compile(buffer)
```
it will behave absolutely identically